src/platform/user/profile/vet360/components/base/Vet360EditModalErrorMessage.jsx

```diff
@@ -54,10 +54,11 @@ export default function Vet360EditModalErrorMessage({
 
   return (
     <AlertBox
-      isVisible
-      status="error"
       content={<div className="columns">{content}</div>}
+      isVisible
       onCloseAlert={clearErrors}
+      scrollOnShow
```

Comment:
does this prop work with the current version of the AlertBox or do we need to wait for your update to formation-react to get published and also do a version bump in our package.json so we are sure to get the latest version of formation-react?

Reply:
I think once the new version is published @sanlouise you may want to run `yarn install` again and add the `yarn.lock` diff 🙂

Reply:
I think we're good on the package.json because of the caret "@department-of-veterans-affairs/formation-react": "^5.2.0", which after running `yarn` would look for the 5.2.2 I specified

Reply:
I'm pretty sure that just running `yarn install` won't grab the latest version of `formation-react` since it will just install whatever `yarn.lock` specifies. So gotta do `yarn upgrade` to change `yarn.lock`. This is why I like being more specific in `package.json` when possible.

Reply:
In some ways life was easier before lock files. Just pin everything in package.json and it was clear what version of everything you needed.

Reply:
The only Yarn command I know is `yarn install` so I always ticket the version in package.json then `yarn install` 🌮

Reply:
I think when installing deps, `yarn.lock` version is only used if there isn't a version bump available as acceptable per the dependency version in `package.json`, so when you do `yarn` and there's a dep upgrade available it will update the `yarn.lock`